Packs/Lumu/Integrations/Lumu/Lumu.py

```diff
@@ -0,0 +1,1092 @@
+
+        response = self._http_request(
+            "POST",
+            f"api/incidents/{lumu_incident_id}/unmute",
+            params=params,
+            json_data=data,
+            headers=headers,
+        )
+
+        return response
+
+    def consult_incidents_updates_through_rest_request(self, offset, items, time):
+        params = assign_params(offset=offset, items=items, time=time, key=self.api_key)
+        headers = self._headers
+
+        response = self._http_request(
+            "GET",
+            "api/incidents/open-incidents/updates",
+            params=params,
+            headers=headers,
+        )
+
+        return response
+
+    def close_incident_request(self, lumu_incident_id, comment):
+        params = assign_params(key=self.api_key)
+        data = {"comment": comment}
+        headers = self._headers
+
+        response = self._http_request(
+            "POST",
+            f"api/incidents/{lumu_incident_id}/close",
+            params=params,
+            json_data=data,
+            headers=headers,
+        )
+
+        return response
+
+
+""" HELPER FUNCTIONS """
+
+
+def get_hmac_sha256(key: Optional[str], comment: str):
+    if key is None:
+        key = ''
+    return hmac.new(key.encode(), comment.encode(), hashlib.sha256).hexdigest()
+
+
+def generate_hmac_sha256_msg(key: Optional[str], comment: str):
+    hmac_code = get_hmac_sha256(key, comment)
+    return f"{comment} hmacsha256:{hmac_code}"
+
+
+def validate_hmac_sha256(key: str, comment_w_hmac: str, separator="hmacsha256:"):
+    """
+
+    :param key:
+    :param comment_w_hmac: Resolved, from: ServiceNow Thu Nov 10 2022 07:56:13 GMT-0800 (PST),
+    close_code: Solution provided, close_notes: ddddddddddddd hmacsha256:
+    a9d7947047c371a10a44d07b381152951c46b0c954f8ddf1eeefc29fd120b8ae
+    :param separator:
+    :return:
+    """
+    result = [piece.strip() for piece in comment_w_hmac.split(separator)]
+    if len(result) != 2:
+        return False
+    comment, hash_mac = result
+    if get_hmac_sha256(key, comment) == hash_mac:
+        return True
+    return False
+
+
+def is_msg_from_third_party(key, comment):
+    if comment and validate_hmac_sha256(key=key, comment_w_hmac=comment):
+        return True
+    return False
+
+
+def add_prefix_to_comment(commnet: Optional[str]) -> str:
+    return f"from XSOAR Cortex {datetime.today():%Y%m%d_%H%M%S} {commnet},"
+
+
+""" COMMAND FUNCTIONS """
+
+
+def retrieve_labels_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    page = args.get("page")
+    items = args.get("items")
+
+    response = client.retrieve_labels_request(page, items)
+    command_results = CommandResults(
+        outputs_prefix="Lumu.RetrieveLabels",
+        outputs_key_field="id",
+        outputs=response,
+        raw_response=response,
+        readable_output=tableToMarkdown('Incidents', response.get('labels', []), headerTransform=pascalToSpace, removeNull=True)
```

Comment:
please verify that the `readable_output` is correct in all the commands
```suggestion
        readable_output=tableToMarkdown('Labels', response.get('labels', []), headerTransform=pascalToSpace, removeNull=True)
```

Reply:
done in this f47ed2c15d6f2c181b37fe28e7a84706396c6845